index.html

```diff
@@ -1876,6 +1876,85 @@ <h3>Examples</h3>
       </pre>
     </section>
   </section>
+  <section>
+    <h2>Background Blur Effect Status</h2>
+    <p>The {{VideoFrameMetadata}} interface exposes the blur state as a property,
+      which allows apps to know the state for every frame. This is important for
+      scenarios where the app must ensure user privacy by never sending an
+      un-blurred frame off of the user's device.</p>
```

Comment:
Mediacapture uses "device" to mean cameras and mics. Maybe:
```suggestion
      un-blurred frame off of the user's system.</p>
```